### PR TITLE
Displayed list of items on the Homepage from the API 

### PR DESCRIPTION
**Pull request by rominana**
In this project I: 

- Displayed list of items on the Homepage. 
- Created the main part of the homepage that keeps the layout from the wireframe:
- When the page loads, the web app retrieves data from the selected API and shows the list of items on the screen with the basic data (e.g. title + image).
- Prepared also "Comments" and "Reservations" buttons.

### CODE REVIEW

File: src/modules/get-shows.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  const response = await fetch(`${tvApi}/show`);
+  const data = await response.json();
+  // we can display more or less shows if we like
+  const shows = data.slice(0, 99);
+  shows.forEach((data) => {
+    if (data.image !== null) {
+      elementInfo.renderCard(data.name, data.image.medium, data.id, data.summary);
+    }
+  });
+  elementInfo.itemCounter();
```
Comment (Nyame-Wolf) [inline src/modules/get-shows.js]: Kindly add an itemCounter method to the getShows function

File: src/modules/card.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// // eslint-disable-next-line import/no-cycle
+import getModalContainer from './card-content.js';
+
+const displayCard = (imgURL, name, itemID, summary) => {
+  getModalContainer(imgURL, name, summary, itemID);
+};
+
```
Comment (Nyame-Wolf) [inline src/modules/card.js]: You can remove this file. I tried running it using npm and it raised an error. Your code works just fine without it

File: src/modules/card-content.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+              <input type="submit" value="Comment" class="btn comments-button" id="submit-form" />
+            </form>
+        </div>
+    </div>
+      `,
+    showCloseButton: true,
+  });
+}
+
+export default getModalContainer;
```
Comment (Nyame-Wolf) [inline src/modules/card-content.js]: Please remove this file too. Your code works just fine without it.

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
Fantastic job so far. The page already looks professionally done and is awesome. There however are a few changes i would like to request you to make

**Comment by rominana** [comment]
All the suggested changes have been applied :smile:

**Review by Nyame-Wolf — APPROVED** [review]
Good job applying the changes. The app looks really nice

**Pull request merged**